svelte/components/Media/MediaPreview.svelte

```diff
@@ -17,8 +17,16 @@
   export let alt: string = nft.name || "media";
 
   let open = false;
-  let metadatas: NftMetadata;
-  $: metadatas = nft?.metadata as NftMetadata;
+  let metadatas: NftMetadata | undefined;
+  $: metadatas = nft?.metadata;
+
+  const handleClose = () => (open = false);
+  const handlemetadatas = () =>
+    !metadatas?.animation_url
```

Comment:
to be refactorized  (at least one bug inside)

Reply:
can you specify witch bug ?